src/main/java/pl/sda/weather/model/WeatherApi.java

```diff
@@ -44,71 +46,45 @@ public int getHumidity() {
             return humidity;
         }
 
-        public float getTemp_min() {
-            return temp_min;
+        public float getWind_speed() {
+            return wind_speed;
         }
 
-        public float getTemp_max() {
-            return temp_max;
-        }
-
-        public float getFeels_like() {
-            return feels_like;
-        }
-
-        public float getSea_level() {
-            return sea_level;
-        }
-
-        public float getGrnd_level() {
-            return grnd_level;
+        public float getWind_deg() {
+            return wind_deg;
         }
 
         @Override
         public String toString() {
             return "MainWeather{" +
-                    "temp=" + temp +
-                    ", temp_min=" + temp_min +
-                    ", temp_max=" + temp_max +
+                    "dt=" + dt +
```

Comment:
try not to use abbreviations because for me temp means temporary

Reply:
it is taken directly from json received from OpenWeather API